### DISCUSSION TITLE
disappearing lines

### DISCUSSION THREAD

**Discussion by WillEhrendreich** [dotnet/fsharp#12790] [Q&A]
Provide the steps required to reproduce the problem:

I don't have much to go on other than using vs2022 and trying to create new lines. 
![VisualStudio2022FSharpDissapearingLines](https://user-images.githubusercontent.com/55286472/152467027-f12b69b7-6b4e-41fd-8d7b-3471e8f352c8.gif)

**Expected behavior**
text editing that involves you being able to see the lines of text. 

**Actual behavior**

text editing with magically disappearing text. 

**Known workarounds**

Sometimes moving the line up and down with the alt+up or down arrow hotkeys will cause it to reappear.. but that's not a great solution.

**Related information**
Windows 10
.NET 6
Visual Studio 2022

**Comment by dsyme** [comment]
I'm going to convert this to a discussion since we don't have a specific repro for this.

**Comment by WillEhrendreich** [comment]
It seems like there's a conflict with an extension I had. I think it was in visual studio power tools.. I can't remember. I should have mentioned it wasn't stopping me anymore when got rid of the problem, because I would have remembered which extension it was.. but it was something that shrunk whitespace or .. something along those lines..

**Reply by WillEhrendreich** [reply]
this is definitely being caused by codelens in vs
